packages/notarization/commitments.go

```diff
@@ -9,30 +9,19 @@ import (
 	"github.com/iotaledger/goshimmer/packages/database"
 	"github.com/iotaledger/goshimmer/packages/ledgerstate"
 	"github.com/iotaledger/goshimmer/packages/tangle"
-	"github.com/iotaledger/hive.go/marshalutil"
 )
 
-var (
-	prevECR = []byte("previousEpochCommitmentRoot")
-)
-
-// EpochCommitment contains the ECR and prevECR of an epoch.
-type EpochCommitment struct {
-	EI EI
-	EC []byte
-}
-
 // Commitment is a compressed form of all the information (messages and confirmed value payloads) of an epoch.
 type Commitment struct {
 	EI                EI
 	tangleRoot        *smt.SparseMerkleTree
 	stateMutationRoot *smt.SparseMerkleTree
 	stateRoot         *smt.SparseMerkleTree
-	prevECR           []byte
+	prevECR           [32]byte
```

Comment:
It should be prevEC thatis equal to Hash(prevEC+ECR+EI)
Also it would be good to introduce type that will hide [32]byte